components/rewards/alert/assets/success.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="42px" height="42px" viewBox="0 0 42 42" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 50.2 (55047) - http://www.bohemiancoding.com/sketch -->
```

Comment:
Can we run all the SVGs when importing to code base through SVGO tool, or https://jakearchibald.github.io/svgomg/ UI - there are a few wasted lines here, as well as polluting ID space.

Reply:
@petemill sure I will do that. Thing is that design is not final and some icons are not final either